.github/workflows/publish.yml

```
@@ -0,0 +1,36 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  publish:
+    environment:
+      name: pypi
+      url: https://pypi.org/p/packaging
```

Since the version is known, render it into the URL.
```suggestion
      url: https://pypi.org/project/packaging/${{ github.ref_name }}
```

---

This is not what [the action's documentation](https://github.com/pypa/gh-action-pypi-publish?tab=readme-ov-file#trusted-publishing) suggests. Does this work?

---

This is not actually related to the action that I created and maintain. This is what gets used in Deployments API and concerns the job, not the action. GH will link whatever you want and this context is available here.
I only made that example generic because it's common to have tags prefixed with `v` and it those projects would show broken URLs unless they convert this value first, which is complexity I didn't want to include into a granular example.

FWIW, a more complete workflow example is linked from the action readme and is hosted on PyPUG in a form of a guide.